src/announce_entry.cpp

```diff
@@ -99,14 +99,14 @@ namespace libtorrent
 		min_announce = min_time();
 	}
 
-	void announce_entry::failed(aux::session_settings const& sett, int retry_interval)
+	void announce_entry::failed(int const tracker_backoff, int const retry_interval)
```

Comment:
would you mind making tracker_backoff be `libtorrent::time_duration` instead of `int`? or at least leave a todo (since I realize this would be a somewhat unrelated improvement over the existing code)